### PR TITLE
Register anshalplayz.is-a.dev

### PR DESCRIPTION
**Pull request by anshalmanoj**
Register anshalplayz.is-a.dev with CNAME record pointing to anshalplayz.github.io.

### CODE REVIEW

File: domains/anshalworld.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "AnshalPlayz",
+           "email": "",
+           "discord": "1095667817417818142"
+        },
+    
+        "record": {
+            "TXT": "anshalplays.github.io/project"
```
Comment (CuteDog5695) [inline domains/anshalworld.json:9]: I think you wanted to use CNAME for this one, also site isnt accessible

File: domains/anshalplayz.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "AnshalPlayz",
+           "email": "",
+           "discord": "1095667817417818142"
+        },
+    
+        "record": {
+            "CNAME": "anshalplayz.github.io"
```
Comment (CuteDog5695) [inline domains/anshalplayz.json:9]: Site isnt accessible

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/anshalplayz.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/AnshalPlayz/project",
+    "owner": {
+        "username": "AnshalPlayz",
+        "email": "<email@address>",
+        "twitter": "<twitter-username>"
```
Comment (CuteDog5695) [inline domains/anshalplayz.json:7]: Please fill out the email and twitter line (if you dont want to give out your email then leave it blank and just fill out the twitter line)

File: domains/anshalworld.json
Comment (CuteDog5695) [inline domains/anshalworld.json:1]: So I just realised that the link in the TXT was spelt wrongly, but you still have to modify the records like the other file (or you can just delete this one)

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**